integration/build/build_test.go

```diff
@@ -197,3 +197,71 @@ func TestBuildWithEmptyLayers(t *testing.T) {
 	resp.Body.Close()
 	require.NoError(t, err)
 }
+
+// #35652
```

Comment:
nit: can you update the description and describe what we're actually testing for (https://github.com/moby/moby/issues/35652)? Having a clear description what we're testing for prevents the original intent from getting lost when tests are refactored